Advanced_Excel/Source/NET/Advanced_Excel.cs

```diff
@@ -706,7 +706,7 @@ public void MssImage_Insert(object ssWorksheet, byte[] ssImageFile, string ssIma
                 range = ws.Cells[ssRowNumber, ssColumnNumber];
             }
 
-            //Util.LogMessage(JsonConvert.SerializeObject(range));
+            Util.LogMessage(JsonConvert.SerializeObject(range));
```

Comment:
@stuart-kiandra line 709 sometimes causes runtime exceptions if left uncommented. We should probably use `#if DEBUG` here

Reply:
Oh yes, I noticed that and set it to be commented out again, but think forgot to add it to the pull request. Thanks for catching it.

Reply:
Oh, sorry, it is there, just have a look at the PR with all changes. :)

Reply:
Hi Hanno, Just following up on this one. Are you waiting for me to use #if DEBUG instead of leave it commented out?